app/main/viewmanager.ts

```diff
@@ -0,0 +1,128 @@
+'use strict';
```

Comment:
The commit message for this commit should be:
```
BrowserView: Add ViewManger for managing multiple View classes.

The ViewManager class makes it easier to manager multiple Views.
```

because the `ViewManager` class is not a wrapper. The summary can be extended.

Reply:
Can you also add more details in commit message how does it handle multiple view
exactly in more detail, like resize etc.?